DirectProgramming/DPC++FPGA/ReferenceDesigns/mvdr_beamforming/README.md

```diff
@@ -214,3 +216,64 @@ The images below show the dataflow in the MVDR beamforming design. The first ima
 <img src="processing_kernels_ideal.png" alt="processing_kernels_ideal" width="800"/>
 <img src="processing_kernels_fake.png" alt="processing_kernels_fake" width="800"/>
 
+### Using Real IO-pipes
+This section describes how to build and run this reference design on a BSP with real IO pipes. The real IO pipes version does **not** work on Windows and requires a specific system setup and BSP.
+
+#### Getting access to the BSP
+TODO
```

Comment:
I assume any instructions on how to set up the card for this test (both SW and HW setup) will be included with the BSP, and so don't belong here?  Do those instructions exist yet?

Reply:
Berni has that document and it is part of the BSP. I will mention that in this section when I get a message from Josh.